src/einundzwanzig.py

```diff
@@ -45,4 +48,39 @@ def episode(update: Update, context: CallbackContext):
     else:
         message = 'Das ist kein gültiges Podcast-Format! Bitte gibt eins der folgenden Formate an: Alle, Interviews, Lesestunde, News, Weg'
 
-    context.bot.send_message(chat_id=update.effective_chat.id, text=message)
\ No newline at end of file
+    context.bot.send_message(chat_id=update.effective_chat.id, text=message)
+
+def getInvoice(amt, shoutout):
+    config.TALLYDATA['satoshi_amount'] = str(amt)
```

Comment:
Using the same object for each request will run into concurrency issues if two users want to send a shoutout at the same time, since the commands are running asynchronously (unlikely that this is going to happen but we should make sure to avoid that problem). You can either create a new object each time the request is sent or clone the existing one.